Client/Frontend/Browser/TopTabCell.swift

```diff
@@ -55,14 +43,11 @@ class TopTabCell: UICollectionViewCell, NotificationThemeable, TabTrayCell, Reus
 
     let closeButton: UIButton = .build { button in
         button.setImage(UIImage.templateImageNamed(ImageIdentifiers.closeTap), for: [])
-        button.tintColor = UIColor.Photon.Grey40
         button.imageEdgeInsets = UIEdgeInsets(top: 15,
                                               left: UX.tabTitlePadding,
                                               bottom: 15,
                                               right: UX.tabTitlePadding)
-        button.layer.shadowOpacity = 0.8
         button.layer.masksToBounds = false
-        button.layer.shadowOffset = CGSize(width: -UX.tabTitlePadding, height: 0)
         button.semanticContentAttribute = .forceLeftToRight
```

Comment:
The shadow for the close button was added but it cleared up afterward so I could delete the code.